### ISSUE TITLE
 'ValueError: not enough values to unpack' in conll2003_reader.py

### ISSUE THREAD

**Issue by ldSidious** [deepmipt/DeepPavlov#1010]
Hello.
I've got train files from http://files.deeppavlov.ai/deeppavlov_data/conll2003_v2.tar.gz
When I'm trying to read them by "conll2003_reader" I see got the follow error:
> deeppavlov/dataset_readers/conll2003_reader.py in parse_ner_file(self, file_name)
>      84                         pos_tags.append(pos)
>      85                     else:
> ---> 86                         token, *_, tag = line.split()
>      87                     tags.append(tag)
>      88                     tokens.append(token)
> 
> ValueError: not enough values to unpack (expected at least 2, got 1)

It looks like a bug.

**Comment by mu-arkhipov** [comment]
Can you please provide the code that produces this error?

**Comment by mu-arkhipov** [comment]
The following code works fine:
```from deeppavlov.core.data.utils import download_decompress
from deeppavlov.dataset_readers.conll2003_reader import Conll2003DatasetReader 
reader = Conll2003DatasetReader()
data_path = '/tmp/conll2003'
download_decompress('http://files.deeppavlov.ai/deeppavlov_data/conll2003_v2.tar.gz', data_path)
dataset = reader.read(data_path)
```

**Comment by ldSidious** [comment]
Unfortunately, the server with my code is no longer available. I tried 'train_evaluate_model_from_config' method from deeppavlov/core/commands/train.py. In config I specified 
"dataset_reader": {
          "class_name": "conll2003_reader",
          "data_path": "/home/ubuntu/data/conll2003/"
      },

where /home/ubuntu/data/conll2003/ is a directory with files test.txt, train.txt, valid.txt.

**Comment by mu-arkhipov** [comment]
It seems that the data you provided has invalid line that cannot be parsed. This issue is resolved in #999 pull request. It is not released at the moment to master branch, however, you can install the dev branch with this fix via command:
`pip install git+https://github.com/deepmipt/DeepPavlov@dev`

**Comment by yoptar** [comment]
Should be fixed in #1015 
Feel free to reopen this issue if the error stays after updating.
